contribs/gmf/less/mobile.less

```diff
@@ -122,12 +125,24 @@ span.twitter-typeahead {
       content: '\f00d';
       color: grey;
       font-family: FontAwesome;
-      font-size: 1.5em;
+      font-size: 2em;
```

Comment:
In my opinion, the icon size is OK. But it's too bold. The idea is to change the icon used.